src/PluginsRegistry.js

```diff
@@ -69,7 +69,7 @@ export default class PluginsRegistry {
   /**
    * Call `onUpdate` hooks.
    */
-  onUpdate(data: Object, rule: Rule, sheet: StyleSheet): void {
+  onUpdate(data: ?Object, rule: ?Rule, sheet: StyleSheet): void {
```

Comment:
where is it comming from? I don't think those can be optional

Reply:
Seems like data argument can be `undefined`. I added checks and integration tests are failed when data is always object.

Reply:
`data` is optional here https://github.com/cssinjs/jss/blob/master/src/RuleList.js#L137